UI/index.html

```diff
@@ -1,37 +1,41 @@
-
 <!Doctype html>
 <html>
+
 <head>
 	<meta charset="UTF-8">
 	<title>{{{config.userName}}}'s Dairy</title>
 	<link href='http://fonts.googleapis.com/css?family=Adamina' rel='stylesheet' type='text/css'>
 	<link href='http://fonts.googleapis.com/css?family=Merriweather:400,300' rel='stylesheet' type='text/css'>
 	<link href='http://fonts.googleapis.com/css?family=Josefin+Sans:300,400' rel='stylesheet' type='text/css'>
 	<link href='http://fonts.googleapis.com/css?family=Raleway:300,400,700' rel='stylesheet' type='text/css'>
+	<link href='http://fonts.googleapis.com/css?family=Berkshire+Swash' rel='stylesheet' type='text/css'>
 	<link rel="stylesheet" href="style.css">
 </head>
-<body>
-	<header>
-		{{{config.userName}}}'s Dairy.
-		<div id="info" contenteditable="true"></div>
-    </header>
-    <h1 contenteditable="true"> 
-        <!-- Removing dropcap for now messing with css.. Can anyone fix this-->
-        <!-- <span id="dropcap">T</span>  -->
-        Title</h1>   
-    <br>
-	<main contenteditable="true">
-		<p>
-                  Hello world.
-		</p>
 
-	</main>
-    <script src="../dropcap.min.js"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
-    <script src="/dairy.js"></script>
-	<!-- <script>
-		var dropcap = document.getElementById("dropcap");
-		window.Dropcap.layout(dropcap, 2);
-	</script> -->
+<body>
+	<section id="headerBar">
+		<header>
+			{{{config.userName}}}'s Dairy.
+			<div id="info" contenteditable="true"></div>
```

Comment:
Also this div id="info" is created for the purpose of date and day data. Since it is not 'sample populated' the user might miss to notice/use it. @uvsankar can you provide the variable name to be populated here for default date and day with the current date and day.

Reply:
@nithinsanjey  yeah will do.